web/themes/custom/pineapple/components/_card.twig

```diff
@@ -25,9 +25,10 @@
       </div>
     {% endif %}
     {% include "@components/_heading.twig" with {
-      heading_level: 2,
+      heading_level: heading_level,
       heading_url: true,
```

Comment:
heading url should be the actual url not a boolean value

Reply:
done